docs/dev_guide/diagnostic_ref/dpct1100.rst

```diff
@@ -28,4 +28,74 @@ If you are running code on a non-GPU device, you can:
 
 * Use ``dpct::select_device(<device_id>)`` to choose a GPU device.
 * Call ``dpct::fft::fft_engine::use_internal_workspace(true)`` before invoking
-  ``dpct::fft::fft_engine::commit()``.
\ No newline at end of file
+  ``dpct::fft::fft_engine::commit()``.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo(cufftHandle plan, float2 *idata, float2 *odata) {
+     size_t worksize;
+     void *workspace;
+     cufftSetAutoAllocation(plan, 0);
+     cufftMakePlan1d(plan, 7, CUFFT_C2C, 2, &worksize);
+     cudaMalloc(&workspace, worksize);
+     cufftSetWorkArea(plan, workspace);
+     cufftExecC2C(plan, idata, odata, CUFFT_FORWARD);
+   }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo(dpct::fft::fft_engine_ptr plan, sycl::float2 *idata,
+            sycl::float2 *odata) {
+     dpct::device_ext &dev_ct1 = dpct::get_current_device();
+     sycl::queue &q_ct1 = dev_ct1.in_order_queue();
+     size_t worksize;
+     void *workspace;
+     /*
+     DPCT1100:0: Currently the DFT external workspace feature in the Intel(R)
+     oneAPI Math Kernel Library (oneMKL) is only supported on GPU devices. Use the
+     internal workspace if your code should run on non-GPU devices.
+     */
+     plan->use_internal_workspace(0);
```

Comment:
Please check if DPCT1100 still hold with 2024.0( or latest MKL behvavior)

Reply:
2024.0 MKL still has this limitation.